UI/data/themes/Rachni.qss

```diff
@@ -0,0 +1,1032 @@
+/***************************************************************************/
+/*   Copyright (C) 2017-2018 by Joel Bethke <joel.bethke@gmail.com>        */
+/*                                                                         */
+/*                                                                         */
+/*   This program is free software: you can redistribute it and/or modify  */
+/*   it under the terms of the GNU General Public License as published by  */
+/*   the Free Software Foundation, either version 2 of the License, or     */
+/*   (at your option) any later version.                                   */
```

Comment:
License:  Is this actually okay?  I know that OBS itself is strictly GPLv2, so derivative works must have a compatible license, and this (in the GPLv2 option) is still compatible...  @dodgepong ?